test/validate.spec.js

```diff
@@ -411,6 +425,43 @@ describe('screener-runner/src/validate', function() {
       });
     });
 
+    describe('validate.sauce', function() {
+      it('should forbid tunnelIdentifier when launchSauceConnect is true', function() {
+        return Validate.runnerConfig({apiKey: 'key', projectRepo: 'repo', states: [], sauce: {username: 'user', accessKey: 'key', launchSauceConnect: true, tunnelIdentifier: 'id'}})
+          .catch(function(err) {
+            expect(err.message).to.equal('child "sauce" fails because [child "tunnelIdentifier" fails because ["tunnelIdentifier" is not allowed]]');
+          });
+      });
+
+      it('should forbid tunnel object when launchSauceConnect is true', function() {
+        return Validate.runnerConfig({apiKey: 'key', projectRepo: 'repo', states: [], tunnel: {host: 'host'}, sauce: {username: 'user', accessKey: 'key', launchSauceConnect: true}})
+          .catch(function(err) {
+            expect(err.message).to.equal('"tunnel" is not allowed');
+          });
+      });
+
+      it('should forbid screener browsers when launchSauceConnect is true', function() {
+        return Validate.runnerConfig({apiKey: 'key', projectRepo: 'repo', states: [], browser: [{browserName: 'chrome'}], sauce: {username: 'user', accessKey: 'key', launchSauceConnect: true}})
+          .catch(function(err) {
+            expect(err.message).to.equal('"browser" is not allowed');
+          });
+      });
+
+      it('should forbid screener browsers(sauce browsers also offered) when launchSauceConnect is true', function() {
+        return Validate.runnerConfig({apiKey: 'key', projectRepo: 'repo', states: [], browser: [{browserName: 'chrome'}, {browserName: 'safari', version: '11.1'}], sauce: {username: 'user', accessKey: 'key', launchSauceConnect: true}})
+          .catch(function(err) {
+            expect(err.message).to.equal('"browser" is not allowed');
+          });
+      });
+
+      it('should allow sauce browsers when launchSauceConnect is true', function() {
+        return Validate.runnerConfig({apiKey: 'key', projectRepo: 'repo', states: [], browsers: [{ browserName: 'chrome', version: '78' }, { browserName: 'firefox', version: '70' }, { browserName: 'internet explorer', version: '11.0' }]})
```

Comment:
@xwa130saucer to use Sauce browsers, they need to have `.` - such as `70.0`

Reply:
Yes, that's right, I forgot to set the flag to be true so the test passed...
When I set the flag as true, it failed. I will update the cases. 
Thanks for pointing out!